src/main/kotlin/no/nav/pensjon/kalkulator/simulering/SimuleringType.kt

```diff
@@ -2,6 +2,7 @@ package no.nav.pensjon.kalkulator.simulering
 
 enum class SimuleringType {
     ALDERSPENSJON,
+    AFP_ETTERF_ALDER,
```

Comment:
Foreslår at denne endres til `PRE2025_OFFENTLIG_AFP_ETTERFULGT_AV_ALDERSPENSJON`.
Dette fordi vi har brukt fullstendige ord for de andre typene.